pylint.rc

```diff
@@ -1,5 +1,6 @@
 [TYPECHECK]
 ignored-classes=SyncManager,_socketobject
+generated-members=py.*
```

Comment:
What does this do?

Reply:
The `py` module seems to have some dynamically generated members and `pylint` complains about not finding the ones used by the module:
```
Module 'py' has no 'path' member (no-member)
```

This is the same issue reported [here](https://github.com/PyCQA/pylint/issues/640). Adding this option to pylint just silences this error.

This will be temporary anyway, as I explain below.